SECURITY.md

```diff
@@ -16,9 +16,40 @@ For example, as of 2024-04-18 the latest release is 0.23.4.
 Therefore 0.23.x, 0.22.x and 0.21.x will be updated, while 0.20.x and 0.19.x
 will not be.
 
-_Note: We use the date of `crates.io` publication when evaluating the security
-policy. For example, while the Rustls 0.20.0 GitHub release note was created
-Jul, 2023 the actual release in `crates.io` was published in Sept. 2021._
+> [!NOTE]
+> We use the date of `crates.io` publication when evaluating the security
+> policy. For example, while the Rustls 0.20.0 GitHub release note was created
+> Jul, 2023 the actual release in `crates.io` was published in Sept. 2021.
+
+### Minimum Supported Rust Version
+
+Our MSRV will be no more recent than 9 versions old, or approximately 12 months.
+
+> [!TIP]
+> At the time of writing, the most recent Rust release is 1.82.0.  That means
+> our MSRV could be as recent as 1.73.0. As it happens, it is 1.71.0.
+
+Our MSRV policy only covers the core library crate: it does not cover tests
+or example code, and is not binding on our dependencies.
+
+We may take _optional_ (but _not_ mandatory) new dependencies on a crate with
+a later MSRV than this policy.
+
+> [!NOTE]
+> This is currently the case for our optional dependency on `zlib-rs`, which
+> has a current MSRV of 1.75.0.
+
+We do not consider MSRV changes to be breaking for the purposes of semver.
+
+Once we reach 1.0.0, we will not make MSRV changes in patch releases.
+
+We will not make MSRV changes to security maintenance branches.
+
+> [!NOTE]
+> For the avoidance of doubt: security maintenance branches exist for each
+> release line _that is not the latest_: so (at the time of writing) 0.22
+> and 0.21 have maintenance branches, but 0.23 does not and is released from
+> the `main` branch.
```

Comment:
Yes, that is correct. To do otherwise would mean MSRV changes are only allowed on a breaking release. (ie, 0.24.0 or 1.0.0 would have to decide the MSRV for that entire release line).